tests/cypress/support/helpers.js

```diff
@@ -34,6 +35,22 @@ function intercept_repair_triage() {
   }).as('repairTriage');
 }
 
+function intercept_get_communal_property_repairs() {
+  const response = mockCommunalRepairs
+
+  cy.intercept('GET', `${api_url}/getCommunalPropertyRepairs?*`, {
```

Comment:
Will need to update this when the update requested by Bhavesh has been done: 
${api_url}/CommunalPropertyRepairs